pegasus/sites.v3/code.org/public/accessibility.md

```diff
@@ -0,0 +1,75 @@
+
+We are working on a holistic strategy to support accessibility throughout our platform. Our vision is to close gaps in access to computer science education for students with disabilities; establish a measure of learning and inclusion; and make improvements to our tools based on this data—while incorporating feedback from our stakeholders to support the learning needs of all students. 
+
+Code.org is committed to continually improving the accessibility of our tools. As of October 2020, Code.org is not yet compliant with [WCAG 2.0 Level A](https://www.w3.org/TR/WCAG20/). In 2021, we are focusing on creating a holistic strategy for addressing accessibility support, as well as investing in high impact improvements we identify along the way. Our long-term plan toward compliance is detailed below.
+
+
+## Our Efforts
+### Hour of Code
+* Align the Code.org Hour of Code experience with [WCAG 2.0 Level A](https://www.w3.org/TR/WCAG20/) compliance.
+* Encourage organizations submitting potential Hour of Code tutorials to list how they have addressed accessibility as part of their submission and make these tutorials and their accessibility features easy to find.
+
+#### Current Accommodations:
+* The Hour of Code tutorials page has a filter to highlight the tutorials from our partners that are accessible via screen-reader.
+
+### K-12 Curriculum
+* Continue partnering with [AccessCSForAll](https://www.washington.edu/accesscomputing/accesscsforall) to provide an accessible version of our [AP CSP curriculum in Quorum](https://quorumlanguage.com/lessons/code.html).
+* Assess ways of making existing and new tools accessible, adopting strategies and technologies shown to be effective.
+* Investigate disaggregation of data on students with disabilities and integrate it into our ongoing data collection and analysis.
+
+#### Current Accommodations:
+* [Microsoft Immersive Reader](https://www.onenote.com/learningtools) is enabled in all Code.org lessons in the instructions panel, offering customization for students to improve reading comprehension and focus.
+        
+<img src="/images/accessibility/MicrosoftImmersiveReaderIcon.png" alt="Microsoft Immersive Reader icon" style="width: 60%; display:block; margin-right: auto; margin-left:auto">
+
+* In-house Text To Speech capabilities are enabled in many lessons in the instructions panel.
+
+<img src="/images/accessibility/TextToSpeechIcon.png" alt="Text to Speech icon" style="width: 60%; display:block; margin-right: auto; margin-left:auto">
+
+* All videos we create are captioned in English by our team or volunteers.
+* Videos used in lessons have show notes allowing students to read a summary of the video content along with still images captured from the video.
```

Comment:
It seems "show notes" should be in quotes, or just be "notes".

Reply:
I spent so long struggling with this phrasing since show notes is both a verb and a noun! :) The quotes will definitely help - thanks!